paket.dependencies

```diff
@@ -48,6 +48,7 @@ group androidapp
   nuget Newtonsoft.Json 11.0.2
   nuget FSharp.Data 3.0.0
   nuget Xamarin.Essentials 1.3.1
+  nuget Xamarin.Forms.PancakeView 1.3.6
```

Comment:
Could you add this package to the `iosapp` group too, please?
Currently, the iOS is crashing because it can't find a dll of PancakeView

You will need to run `.paket\paket.exe install` then (or `mono .paket\paket.exe install` on macOS)
This will update the paket.lock file